### DISCUSSION TITLE
ERROR on build

### DISCUSSION THREAD

**Discussion by alienatorZ** [iTowns/itowns#2001] [Q&A] [answered]
I am gettting the following error `
ERROR in ./node_modules/@tmcw/togeojson/dist/togeojson.umd.js 1:366
Module parse failed: Unexpected token (1:366)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
`

Does anyone know how to resolve?

**Comment by jailln** [comment]
Hi,
Could you describe the context where you get this error please ?

**Reply by alienatorZ** [reply]
I am building a visualization application in react. I want to use Itowns as the mapping application. My project uses webpack and I used `npm install --save itowns` to import the library. When I ran `npm run build` I get that error. It seems related to code in togeojson in node_modules.

**Reply by alienatorZ** [reply]
Is there an example webpack itowns application I could reference? Maybe I need some sort of loader to handle the togeojson.umd.js code.

**Reply by jailln** [reply]
Normally you shouldn't need anything specific, you can take a look at [UD-Viz](https://github.com/VCityTeam/UD-Viz/tree/master/packages/browser) which is packaged with webpack and imports itowns. Let me know if that's of any help.

**Comment by alienatorZ** [comment, marked as answer]
I could never get it to work. I had to use the already built itowns.js file. I started getting THREE.Vector3 is not a constructor error again. using the itowns.js cleared up the errors.

**Reply by mgermerie** [reply]
Hi ! Would you have a link to your project to share ? I tried setting up a clean repo where I installed iTowns and buffer (which should be added as a peer dependency of iTowns) with npm. Running `npx webpack` from there gave me no error. If I could take a look at your `webpack.config` and `package.json`, it would help me figure out where the issue comes from. You can mail me this content if you do not wish to share your project publicly : madec.germerie-guizouarn@ign.fr.

**Reply by alienatorZ** [reply]
It looks like we are switching to vite which may make this question obsolete. Thank you for your help and time though. I will try with Vita and see if I get different results.
